Popup/tests/Popup-specs.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
+import {mount} from 'enzyme';
+
+import {Popup} from '../Popup';
+
+const FloatingLayerController = FloatingLayerDecorator('div');
+
+describe('Popup specs', () => {
+	test('should be rendered opened if open is set to true', () => {
+		const popup = mount(
+			<FloatingLayerController>
+				<Popup open>
+					<div>popup</div>
+				</Popup>
+			</FloatingLayerController>
+		);
+
+		const expected = true;
+		const actual = popup.find('FloatingLayer').prop('open');
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should not be rendered if open is set to false', () => {
+		const popup = mount(
+			<FloatingLayerController>
+				<Popup>
+					<div>popup</div>
+				</Popup>
+			</FloatingLayerController>
+		);
+
+		const expected = false;
+		const actual = popup.find('FloatingLayer').prop('open');
+
+		expect(actual).toBe(expected);
+	});
+});
```

Comment:
Can you add at least one test for the `center` prop because you add that prop in this PR?